orb.yml

```diff
@@ -11,19 +11,104 @@ commands:
+        unzip apache-groovy-binary.zip
+        mv groovy-<<parameters.groovy-version>> groovy
 
   archive:
+    description: "Archives artifacts to Nexus Repository Manager"
+    parameters:
+      username:
+        description: "Username to access Nexus Repository Manager"
+        type: string
+        default: "${NEXUS_RM_USERNAME}"
+      password:
+        description: "Password to access Nexus Repository Manager"
+        type: string
+        default: "${NEXUS_RM_PASSWORD}"
+      serverurl:
+        description: "Nexus Repository Manager Server URL"
+        type: string
+        default: "${NEXUS_RM_SERVERURL}"
+      filename:
+        description: "Path of file to publish"
+        type: string
+      format:
+        description: "Format of artifact to publish"
+        type: string
+        default: "maven2"
+      attributes:
+        description: "Component coordinates and Asset attributes"
+        type: string
+      repository:
+        description: "Target repository name"
+        type: string
+        default: "maven-releases"
+
     steps:
-    - run: echo "Archive command will start here"
+    - attach_workspace:
+        at: /tmp/workspace
+    - run: >
+        groovy/bin/groovy NexusPublisher.groovy
+        --username <<parameters.username>>
+        --password <<parameters.password>>
+        --serverurl <<parameters.serverurl>>
+        --filename <<parameters.filename>>
+        --format <<parameters.format>>
+        --repository <<parameters.repository>>
+        <<parameters.attributes>>
 
 jobs:
-  nexus-platform:
+  nexusjob:
+    description: "Publish artifacts to Nexus Respository Manager"
+    parameters:
+      username:
+        description: "Username to access Nexus Repository Manager"
+        type: string
+        default: "${NEXUS_RM_USERNAME}"
+      password:
+        description: "Password to access Nexus Repository Manager"
+        type: string
+        default: "${NEXUS_RM_PASSWORD}"
+      serverurl:
+        description: "Nexus Repository Manager Server URL"
+        type: string
+        default: "${NEXUS_RM_SERVERURL}"
+      filename:
+        description: "Path of file to publish"
+        type: string
+      format:
+        description: "Format of artifact to publish"
+        type: string
+        default: "maven2"
+      attributes:
+        description: "Component coordinates and Asset attributes"
+        type: string
+      repository:
+        description: "Target repository name"
+        type: string
+        default: "maven-releases"
+      groovy-version:
+        description: "Version of Apache groovy to download and install"
+        type: string
+        default: "2.5.3"
     executor: default
     steps:
-    - install
+    - install:
+      groovy-version: << parameters.groovy-version >>
+    - archive:
+        username: << parameters.username >>
```

Comment:
Very minor: The parameter value spacing in `<< >>` is different between this `archive` block and the `run` block above.

Reply:
make <<>> consistent